Dockerfile

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) Sameer1046
 # SPDX-License-Identifier: AGPL-3.0-only
+
 # Build the image from source
 FROM gradle:6.8.2-jdk11 AS build
 COPY --chown=gradle:gradle . /home/gradle/src
 
 WORKDIR /home/gradle/src
-
 RUN gradle build --no-daemon --exclude-task test
 
 
-#Create the containerized app
-FROM adoptopenjdk/openjdk11:jre-11.0.15_10-ubuntu
+# Create the containerized app
+FROM openjdk:11-jre-slim-buster
```

Comment:
Smaller Docker images take up less disk space. The [openjdk:11-jre-slim-buster](https://hub.docker.com/layers/library/openjdk/11-jre-slim-buster/images/sha256-402f01eb4ceff5909e29f90a9fb2e897b7a2ae00151777246f3efb5c9b37671f?context=explore) image is only **72.92 MB**.